### PR TITLE
Feature/profile dropdown

### LINKED ISSUE #34: Add profile dropdown
- [x] should link 
  - [x] edit page
  - [x] profile page #30

### PR DESCRIPTION
**Pull request by konradh**

### CODE REVIEW

File: app/views/layouts/_navbar.html.erb [hunk 1]
```diff
@@ -14,8 +14,15 @@
...
             <%# https://github.com/heartcombo/devise#controller-filters-and-helpers %>
             <% if user_signed_in? %>
-            <li class="nav-item">
-                <%= link_to (fa_icon 'user', text: current_user.email), edit_user_registration_path, class: 'nav-link'%>
+            <li class="nav-item dropdown">
+              <a class="nav-link dropdown-toggle" href id="navbarProfileDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+                  <%= current_user.username %>
+                </a>
+                <div class="dropdown-menu" aria-labelledby="navbarProfileDropdown">
+                  <%= link_to "Visit my profile", user_path(current_user), class: 'dropdown-item' %>
```
Comment (sirkrypt0) [inline app/views/layouts/_navbar.html.erb]: Use something like the following for internationalization. Make sure to set the corresponding value in `config/locales/en.yml`.

```suggestion
                  <%= link_to I18n.t('menu.visit_profile'), user_path(current_user), class: 'dropdown-item' %>
```

File: app/views/layouts/_navbar.html.erb [hunk 2]
```diff
@@ -14,8 +14,15 @@
...
+                  <div class="dropdown-divider"></div>
+                  <%= link_to "Edit my profile", edit_user_registration_path(current_user), class: 'dropdown-item' %>
```
Comment (sirkrypt0) [inline app/views/layouts/_navbar.html.erb]: Add internationalization here as well.

**Review by sirkrypt0 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 3]
```diff
@@ -71,3 +71,4 @@ capybara-*.html
 
 # Ignore editor configs
 /.vscode
+/.idea
```
Comment (frcroth) [inline .gitignore]: See https://github.com/hpi-swt2/connections-portal/pull/56#issuecomment-738951815

**Review by sirkrypt0 — APPROVED** [review]

**Review by felixauringer — APPROVED** [review]

File: .gitignore (hunk 3 above)
Comment (frcroth) [inline .gitignore:74]:
```suggestion
/.vscode
/.idea
```

Comment (frcroth) [inline .gitignore]:
```suggestion
```

**Pull request merged**